### DISCUSSION TITLE
creating T_b by stiching of several time intervals

### DISCUSSION THREAD

**Discussion by Felixkx** [TDAmeritrade/stumpy#1027] [Help: Best Practices]
Hi,
for creating a similarity index of a time series, I want to use stumpy.stump to find the smallest distance to specific epoques ( can also be in another time series). Thus I want to create a T_B which is a combination of these small epoques. What is the best practice to do this? Is adding a np.nan at the end of each subsequence before stiching them together a good way to do that?

**Comment by seanlaw** [comment]
@Felixkx Thank you for your question and welcome to the STUMPY community. 

> Is adding a np.nan at the end of each subsequence before stiching them together a good way to do that?

Yes, this is the standard approach as it prevents a best match from crossing boundaries. However, you will need to offset the matrix profile index accordingly as everything has shifted when you added the `np.nan`
